src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1475,4 +1481,27 @@ private void ComputePackageExclusions()
             }
         }
     }
+
+    internal static class NuGetTargetFrameworkUtils
+    {
+        public static NuGetFramework GetTargetFramework(string targetFrameworkMoniker, string targetPlatformMoniker)
```

Comment:
For all tasks that make use of the assets file, ideally instead of looking it up by framework, you'd look up by the alias.